### PR TITLE
(DOCSP-9355): Port GitHub Service Snippets

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
JIRA: https://jira.mongodb.org/browse/DOCSP-9355

Stage: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/DOCSP-9355/services/snippets/github.html

### CODE REVIEW

File: source/services/snippets/github.txt [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+If your app does not have a GitHub Service interface, :doc:`create one
+</services/create-a-service>` before using these snippets.
+
+.. contents:: GitHub Snippets
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Log All Commits In MongoDB
```
Comment (cbush) [inline source/services/snippets/github.txt]:
```suggestion
Log All Commits in MongoDB
```

File: source/services/snippets/github.txt [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+--------------------------
+
+.. _GitHub-PushEvent: https://developer.github.com/v3/activity/events/types/#pushevent
```
Comment (nlarew) [inline source/services/snippets/github.txt]: Can we move this to be an inline link? We only use the URL once and this type of link is not supported in NextGen.

File: source/services/snippets/github.txt [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+       console.log(insertResult.insertedIds);
+     } catch(err) {
+       console.error(err)
+     }
+   };
+
+Automatically Comment On New Pull Requests
+------------------------------------------
+
+.. _GitHub-PullRequestEvent: https://developer.github.com/v3/activity/events/types/#pullrequestevent
```
Comment (nlarew) [inline source/services/snippets/github.txt]: Can we move this to be an inline link? We only use the URL once and this type of link is not supported in NextGen.

File: source/services/snippets/github.txt [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+This :ref:`GitHub incoming webhook <github-incoming-webhooks>` function
+adds a comment to new pull requests that thanks users for submitting.
+The webhook accepts a `PullRequestEvent <GitHub-PullRequestEvent>`_
+payload from GitHub and uses an :doc:`HTTP service </services/http>`
+client to :github-dev:`create a comment
```
Comment (nlarew) [inline source/services/snippets/github.txt]: Need to add `github-dev` to `conf.py`

**Pull request merged**